public/javascripts/items.js

```diff
@@ -79,21 +77,54 @@ const completedClicked = event => {
     .then( result => result.json() )
     .then( checkJsonForSuccessField )
     .then( json => {
-      const parent = $( `.title[data-id=${id}]` )
+      const parent = $( `.item__title[data-id=${id}]` )
         element.data( 'completed', completed )
         if( completed ) {
-          parent.addClass( 'completed' )
+          parent.addClass( 'item__title--completed' )
         } else {
-          parent.removeClass( 'completed' )
+          parent.removeClass( 'item__title--completed' )
         }
       }
     )
   }
 
+  const getFilterStatus = () => {
+    const pageURL= decodeURIComponent(window.location.search)
+    const parameterMatch = /completed_filter=(.*)/
+    let filterStatus = parameterMatch.exec(pageURL)
```

Comment:
We should not need these three lines, if we pass the request object (which should expose the query string and its keys).  We should also avoid the use of `let` unless we must mutate the value (we should avoid mutation at all costs).